bin/tigmint-make

```diff
@@ -9,17 +9,37 @@
 # To run Tigmint and calculate assembly metrics:
 # tigmint-make draft=myassembly reads=myreads ref=GRCh38 G=3088269832
 
+# To run Tigmint with long reads:
+# tigmint-make tigmint-long draft=myassembly reads=myreads span=auto gsize=genomesize
+
 # Name of the draft assembly, draft.fa
 draft=draft
 
-# Name of the reads, reads.fq.gz, after running longranger basic
+# Name of the reads, reads.fq.gz, after running longranger basic (for linked reads)
+# Can be reads.fq.gz or reads.fa.gz for long reads
 reads=reads
 
+# Find the complete read file name
+fastq=$(shell test -f $(reads).fq.gz && echo "true")
+fasta=$(shell test -f $(reads).fa.gz && echo "true")
+ifeq ($(fastq), true)
+lr_filename=$(reads).fq.gz
+endif
+ifeq ($(fasta), true)
+lr_filename=$(reads).fa.gz
+endif
+
 # Reference genome, ref.fa, for calculating assembly contiguity metrics
-#ref=ref
+ref=ref
 
 # Size of the reference genome, for calculating NG50 and NGA50
-#G=0
+G=G
+
+# Haploid size of the draft assembly organism's genome, for calculating span
+gsize=-1
```

Comment:
Could these really just be one parameter? Since you'd be using a reference genome of the same size as the draft assembly genome.